src/Vault.sol

```diff
@@ -86,11 +91,30 @@ contract Vault {
     modifier onlyFactory() {
         require(
             msg.sender == IMainRegistry(_registryAddress).factoryAddress(),
-            "VL: Not factory"
+            "VL: You are not the factory"
```

Comment:
Add positive and negative test

Reply:
This is about the modifier or the transfer Owner / liquidate methods?

Reply:
Done except packed slot issue. Created a issue for this